installer-config.txt

```diff
@@ -8,7 +8,7 @@ userpw=raspberry
 boot_volume_label=openHABian
 disable_root=1
 user_is_admin=1
-username=pi
+username=admin
```

Comment:
Exactly. You've introduced the non-static username use, which is the important part 👍